Dockerfile

```diff
@@ -27,7 +25,8 @@ RUN apt-get install -yq --no-install-recommends pkg-config
 RUN apt-get install -yq --no-install-recommends zlib1g-dev
 RUN apt-get install -yq --no-install-recommends libbz2-dev
 RUN apt-get install -yq --no-install-recommends unzip
-RUN apt-get install -yq --no-install-recommends libpng12-dev
+RUN apt-get install -yq --no-install-recommends libpng-dev
+RUN DEBIAN_FRONTEND="noninteractive" apt-get install -yq --no-install-recommends tzdata
```

Comment:
I'd just put that before the apt-gets rather than embed in the command, as you have in the second phase of the build